_includes/footer.html

```diff
@@ -5,6 +5,11 @@
 
       <footer>
         <ul class="list-inline">
+          {% if page.path contains 'docs' %}
```

Comment:
Nice change. But I think this is better done through our build script in the main repo. We already add things like version added and such in that script. Adding a link to edit the page should be pretty easy there, and wouldn't require string comparison.